native/CMakeLists.txt

```diff
@@ -3028,7 +3134,7 @@ if(ANDROID)
     )
 endif()
 
-if(OHOS)
+if(NOT OPENHARMONY AND OHOS)
```

Comment:
OpenHarmony also defines `OHOS`?

Reply:
Yes, OHOS is defined in both openharmony and HarmonyOS